Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16.20.2-alpine3.18
+FROM node:20.11.1-alpine3.18
```

Comment:
can we add an "AS" block here so this line will be:

 FROM node:20.11.1-alpine3.18 AS fnl_base_image

This is needed for ongoing security scanning